### DISCUSSION TITLE
what is the relationship between Tailwind css and windi css?

### DISCUSSION THREAD

**Discussion by JenuelDev** [windicss/windicss#209] [Q&A] [answered]
they work the same, the new JIT was already a feature on windi css. I just want to know the relationship and what is better to use?

**Comment by antfu** [comment, marked as answer]
https://github.com/windicss/windicss/discussions/176

**Reply by JenuelDev** [reply]
oh I see, thanks thanks,, im actually thinking what to choose between the two.. and thanks for the reply, I believe windi CSS is worth trying.. :)
